websites/S/Sorryfield/presence.ts

```diff
@@ -0,0 +1,156 @@
+const presence = new Presence({
+	clientId: "1016312551958642698",
+});
+
+const enum Assets {
+	LOGO = "large-image",
+	CHART_SEARCH = "chart-search",
+	CHART_HYPORANKED = "chart-hyporanked",
+	CHART_EXPONENTIAL = "chart-exponential",
+	CHART_HARD = "chart-hard",
+	CHART_INTERMEDIATE = "chart-intermediate",
+	CHART_EASY = "chart-easy",
+	CHART_HISTORY = "chart-history",
+	CHART_POSSESION = "chart-possesion",
+	CHART_HOT = "chart-hot",
+	CHART_COLD = "chart-cold",
+	CHART_MULTIPLAYER = "chart-multiplayer",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		details: "Sorryfield",
+		smallImageKey: Assets.LOGO,
+	};
+	if (
+		document.location.pathname === "/" ||
+		document.location.pathname === "/sorrygle" ||
+		document.location.pathname.startsWith("/song/")
+	) {
+		presenceData.largeImageKey = Assets.LOGO;
+		presenceData.smallImageKey = null;
+	}
+	if (document.location.pathname === "/")
+		presenceData.details = "Viewing Homepage";
+	if (document.location.pathname === "/sorrygle")
+		presenceData.details = "Sorrygle";
+	if (document.location.pathname.startsWith("/song/")) {
+		presenceData.details = "Listening the song";
+		const songName = document
+			.querySelector(".title")
+			.innerHTML.replace('<i class="icon fa-fw fas fa-music"></i>&nbsp;', "")
+			.replace('<i class="icon fa-fw fas fa-lock"></i>&nbsp;', "")
+			.replace('<span class="desc">&nbsp;', " ")
+			.replace("</span>", "")
+			.replace("&amp;", "&");
+		let songArtist = document
+			.querySelector(".song")
+			.children[2].innerHTML.split("&nbsp;")[0];
+		if (document.querySelector(".title").innerHTML.includes("fa-lock"))
+			songArtist = `🔒 ${songArtist}`; // locked music
+		presenceData.state = `${songArtist} - ${songName}`;
+		presenceData.buttons = [
+			{ label: "Listen the song", url: document.location.href },
```

Comment:
Button text should be capitalized.
```suggestion
			{ label: "Listen To The Song", url: document.location.href },
```
```suggestion
			{ label: "Listen", url: document.location.href },
```